tests/testapp/__init__.py

```diff
@@ -1,2 +0,0 @@
-# -*- coding: utf-8 -*-
-
```

Comment:
I thought that this file is necessary, but it seems that there is no problem because the test is passed...